server/app/views/admin/programs/ManageProgramAdminsView.java

```diff
@@ -64,72 +71,105 @@ public Content render(
+    FieldWithLabel emailField =
+        FieldWithLabel.email()
+            .setId("admin-email-input")
+            .setFieldName("adminEmail")
+            .setLabelText("Admin email address");
+    return div()
+        .with(
+            formTag.with(
+                emailField.getInputTag(),
+                makeCsrfTokenInputTag(request),
+                submitButton("Add")
+                    .withId("add-admin-button")
+                    .withClasses(ButtonStyles.SOLID_BLUE, "ml-2", "mb-6")))
+        .withClasses("border", "border-gray-300", "shadow-md", "w-1/2", "mt-6");
+  }
+
+  private DivTag renderExistingAdmins(
       Http.Request request, long programId, ImmutableList<String> existingAdminEmails) {
-    DivTag emailFields =
-        div()
-            .withId(EMAIL_CONTAINER_DIV_ID)
-            .withClasses("ml-4")
-            .with(each(existingAdminEmails, email -> adminEmailInput(Optional.of(email))))
+    return div(
+        table()
+            .withClasses("border", "border-gray-300", "shadow-md", "w-full")
+            .with(renderExistingAdminsTableHeader())
             .with(
-                ViewUtils.makeSvgTextButton("Add admin", Icons.PLUS)
-                    .withType("button")
-                    .withId(ADD_BUTTON_ID)
-                    .withClasses(ButtonStyles.OUTLINED_WHITE_WITH_ICON, "my-2"));
+                tbody(
+                    each(
+                        existingAdminEmails,
+                        email -> renderExistingAdminRow(request, programId, email)))));
+  }
 
-    return form()
-        .with(makeCsrfTokenInputTag(request))
-        .withAction(routes.ProgramAdminManagementController.update(programId).url())
-        .withMethod("POST")
-        .with(emailFields)
-        .with(submitButton("Save").withClasses(ButtonStyles.SOLID_BLUE, "my-4"));
+  private TheadTag renderExistingAdminsTableHeader() {
+    return thead(
+        tr().withClasses("border-b", "bg-gray-200", "text-left")
+            .with(th("Admin email").withClasses(BaseStyles.TABLE_CELL_STYLES, "w-2/3"))
+            .with(
+                th("Actions")
+                    .withClasses(BaseStyles.TABLE_CELL_STYLES, "text-right", "pr-8", "w-1/3")));
   }
 
-  private DivTag adminEmailInput(Optional<String> existing) {
-    // When there are existing admins, the only option is to remove that admin. The field is
-    // disabled, so that no changes except removal can be made. The form does not submit disabled
-    // fields, so these existing admins will not be removed unless the remove button is clicked,
-    // which sets disabled to false (see TypeScript file).
-    String inputFieldName = existing.isPresent() ? REMOVE_EMAIL_FIELD_NAME : ADD_EMAIL_FIELD_NAME;
+  private TrTag renderExistingAdminRow(Http.Request request, long programId, String adminEmail) {
+    return tr().withClasses(
+            ReferenceClasses.ADMIN_TI_GROUP_ROW,
```

Comment:
TI here means Trusted Intermediary. Since this table is not related to TIs, I don't think we should use this reference class.